backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -2226,13 +2226,26 @@ class PathSuffixSchema(marshmallow.Schema):
 class UserMessagesMarkAsReadQuerySchema(marshmallow.Schema):
     content_ids = StrippedString(
         validate=regex_string_as_list_of_int,
-        example="3,4",
-        description="comma separated list of content_ids to check for marking event as read",
+        example="1,4",
+        description="Comma separated list of content ids. Every event related to these contents\
+            will be marked as read.",
+    )
+    notification_ids = StrippedString(
```

Comment:
```suggestion
   event_ids = StrippedString(
```
Notifications do not exist for the backend.